test/dag-node-test.js

```diff
@@ -29,10 +29,10 @@ module.exports = (repo) => {
         expect(Buffer.isBuffer(node.data)).to.be.true.mark()
         expect(node.size).to.be.above(0).mark()
 
-        util.serialize(node, (err, serialized) => {
+        DAGNode._serialize(node, (err, serialized) => {
```

Comment:
@haadcode it does make sense because `util` and `resolver` is what an IPLD Resolver needs in order to process an IPLD Format, this is generic to any API resolver.